protocol/testing/genesis.sh

```diff
@@ -1476,7 +1476,7 @@ function update_genesis_use_test_exchange() {
 	# All remaining markets can just use the LINK ticker so the daemon will start. All markets must have at least 1
 	# exchange. With only one exchange configured, there should not be enough prices to meet the minimum exchange
 	# count, and these markets will not have index prices.
-	for market_idx in {3..33}
+	for market_idx in {3..34}
```

Comment:
dydx token was not accounted for here when I added it a few weeks ago, this caused problems because it mentions non-existent markets as adjustment markets. Was not a problem before because we had only enabled the test exchange for container tests.

RN, container tests are running a daemon with all exchanges. I can update this once I add a flag that allows users to disable exchanges.

Reply:
Could run `make update-sample-pregenesis` to check if prod_sample_genesis is changed as a result of this?

Reply:
No updates!